src/main/java/step2/LottoMain.java

```diff
@@ -0,0 +1,10 @@
+package step2;
+
+import step2.controller.LottoController;
+
+public class LottoMain {
+    public static void main(String[] args) {
+        LottoController lottoService = new LottoController();
+        lottoService.buyManualLotto();
```

Comment:
```suggestion
        LottoController lottoController = new LottoController();
        lottoController.buyManualLotto();
```

깜빡하고 수정을 빠트리신것 같네요 ㅎㅎ